temperature@fevimu/files/temperature@fevimu/settings-schema.json

```diff
@@ -22,6 +22,20 @@
     "tooltip": "Whether or not show the unit letter (C or F).",
     "dependency": "show-unit"
   },
+  "show-label-prefix": {
+    "type": "switch",
+    "default": false,
+    "description": "Show label prefix",
+    "tooltip": "Whether or not to add a label before the temperature value."
+  },
+  "label-prefix": {
+    "type": "entry",
+    "default": "🌡 ",
+    "description": "Label prefix",
+    "indent": true,
+    "tooltip": "The label to be shown before the temperature value.",
+    "dependency": "show-label-prefix"
+  },
```

Comment:
Please remove the "indent" line (line 35) and the blank space of the "default" field.